source/javascripts/_componentRegister.js

```diff
@@ -74,4 +74,4 @@ angular
 	)
 	.component("rWorkflowRecipesInfoBanner", register(WorkflowRecipesInfoBanner, []))
 	.component("rProductTour", register(ProductTour, ["menuIds", "currentUser", "productTourShown", "onDismiss"]))
-	.component("rGuidedOnboarding", register(GuidedOnboarding, ["isEnabled", "isOpen", "onTurnOff", "buildStatus"]));
+	.component("rGuidedOnboarding", register(GuidedOnboarding, ["isEnabled", "isOpen", "onTurnOff", "buildStatus", "lastWorkflowEditedApp", "lastWorkflowEditedDate", "lastRunningBuildFinishDate"]));
```

Comment:
It doesn't seem like we're using `lastWorkflowEditedApp` here anywhere. Am I missing something?

Reply:
+1

Reply:
Just to be clear: the approach described on the Jira issue calls for the use of `lastWorkflowEditedApp`, and I don't think you can meet this acceptance criteria without it:

> When the user has saved a workflow and then a build **for the same app** has completed successfully, the icon is a fully filled circle.

I wanted you to use it, not take it out! :)